src/routes/JobViewRoute.js

```diff
@@ -28,20 +33,64 @@ class JobViewRoute extends React.Component {
     }).isRequired,
   };
 
+  constructor(props) {
```

Comment:
We prefer the `constructor`-less approach to initing `state` when you don't actually need the constructor.